nipype/info.py

```diff
@@ -169,7 +169,7 @@ def get_nipype_gitversion():
     'profiler': ['psutil>=5.0'],
     'duecredit': ['duecredit'],
     'xvfbwrapper': ['xvfbwrapper'],
-    'pybids': ['pybids==0.6.5'],
+    'pybids': ['pybids==0.7.0'],
```

Comment:
Do we want to make this such a hard pin? Admittedly, strict pins in extras don't have much force.

Reply:
No I guess we can do `>`